src/components/HistoricalRecordSelector/index.tsx

```diff
@@ -350,19 +350,32 @@ export function HistoricalRecordSelector<T extends BaseRecord>({
           <SheetHeader className="p-0">
             <SheetTitle className="text-lg font-medium">{title}</SheetTitle>
           </SheetHeader>
-          <Tabs
-            value={activeType}
-            onValueChange={handleTabChange}
-            className="w-full"
-          >
-            <TabsList className="w-full">
+          {structuredTypes.length > 1 ? (
+            <Tabs
+              value={activeType}
+              onValueChange={handleTabChange}
+              className="w-full"
+            >
+              <TabsList className="w-full">
+                {structuredTypes.map(({ type }) => (
+                  <TabsTrigger key={type} value={type} className="flex-1">
+                    {type}
+                  </TabsTrigger>
+                ))}
+              </TabsList>
+            </Tabs>
+          ) : (
+            <div className="w-full">
               {structuredTypes.map(({ type }) => (
-                <TabsTrigger key={type} value={type} className="flex-1">
+                <div
+                  key={type}
+                  className="flex-1 cursor-default text-center py-1 text-sm text-black bg-white border border-gray-200 rounded-md shadow-sm"
```

Comment:
why to show it as button or tab trigger if it is not ?


```suggestion
                  className="flex-1 font-semibold text-center py-1 text-md"
```
after suggestion changes:

<img width="764" alt="Screenshot 2025-06-16 at 5 10 29 PM" src="https://github.com/user-attachments/assets/947b1c3c-4032-44d4-8428-8043f4f4bf31" />

Reply:
Why do we need the title? Can't we just concat it with the title as `Symptoms History` 🤔

Reply:
@Sreyas62  👆

Reply:
![Screenshot From 2025-06-19 11-06-24](https://github.com/user-attachments/assets/71a25a62-87ef-4733-9567-5357cac1cd57)
Should be like this?
@bodhish

Reply:
No , see on top left we can directly have title there instead of `History` we should have "Symptom History" in given example

Reply:
![Screenshot From 2025-06-19 11-23-17](https://github.com/user-attachments/assets/8ac45a46-8bc2-4fa8-8fdc-df11e3505f1d)

Reply:
- [ ] Position `Symptom History` to centre 
- [ ]  As bodhi mentioned in comment, don't need title (Symptoms) anymore,  Sheet title is enough